### ISSUE TITLE
Questions about MC Dropout in the code😥

### ISSUE THREAD

**Issue by plusnli** [xiaoyuxin1002/UQ-PLM#1]
Appreciate your work and I have a question about MC Dropout.

I see your code and it seems that when using MC dropout, you first feed the pretrained transformers with data to get the batch_reps and then feed the BNN(VERSION_STO) with the batch_reps and use MC Dropout here.

My questions are:
1) If you do this, what you get is the uncertainty of the **BNN alone** or the  **whole model** ?
2) If I apply MC Dropout to a single pretrained transformer model instead of BNN, does calcuating the mean or variance to present the uncertainty make sense?  I mean, I am confused that if the model is not bayesian model, can what you get using MC Dropout present the uncertainty?(cause the parameters of each neuron is not a distribution, directly using MC Dropout might not make sense?)

I would really appreciate your kind help.🙂

**Comment by xiaoyuxin1002** [comment]
Thanks for your interest in our code!
I think that you might understand the code wrongly.
When using MC dropout, I feed data to the pre-trained transformers without any BNN layers and with dropout turned on.
You can check Line 155 in test.py. I used deterministic models there, meaning no BNN layers.
